install.sh

```diff
@@ -17,13 +17,18 @@ fi
 
 # Remove previous version and setup folder
 echo "Removing previous version (if found)..."
-mkdir -p ~/.local/share/nautilus-python/extensions
-rm -f ~/.local/share/nautilus-python/extensions/VSCodeExtension.py
-rm -f ~/.local/share/nautilus-python/extensions/code-nautilus.py
+if [ $(id -u) == 0 ]; then
+  EXT_DIR=/usr/share/nautilus-python/extensions
+else
+  EXT_DIR=~/.local/share/nautilus-python/extensions
+fi 
+mkdir -p $EXT_DIR
+rm -f $EXT_DIR/VSCodeExtension.py
+rm -f $EXT_DIR/code-nautilus.py
```

Comment:
If you installed the extension before as normal user, and now as root, you'll end up with 2 extensions installed because its not deleting the previous one.

That being said I think I would rather like to avoid doing this to the script, but instead setup a per distro package for the extension that does it instead.

Reply:
Opsie! I didn't notice that. Do you mind if I create a AUR package to install it globally?

Reply:
I would love it if you could do that, its on my TODO list but ever since Antergos got shut down I haven't been using Arch based distros at all.